examples/tutorials/research/02_advanced_usage_of_research.ipynb

```diff
@@ -0,0 +1,1023 @@
+{
```

Comment:
<p>Explain in more detail and with some examples why we need branches</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>